src/main/kotlin/org/jetbrains/intellij/tasks/RunPluginVerifierTask.kt

```diff
@@ -336,19 +329,19 @@ open class RunPluginVerifierTask @Inject constructor(
                     "Provide pre-downloaded IDEs stored in `downloadDir` or use `localPaths` instead."
             ))
             else -> {
-                val ideArchive = File(downloadDir.get(), "${name}.tar.gz")
                 val url = resolveIdeUrl(type, version, buildType)
-
                 debug(context, "Downloading IDE from $url")
 
-                downloadAction.apply {
-                    src(url)
-                    dest(ideArchive.absolutePath)
-                    tempAndMove(true)
-                    execute()
+                val repository = repositoryHandler.ivy { ivy ->
+                    ivy.url = URI(url)
+                    ivy.patternLayout { it.artifact("") }
+                    ivy.metadataSources { it.artifact() }
                 }
+                val dependency = dependencyHandler.create("com.jetbrains:ides:$type-$version-$buildType@tar.gz")
 
                 try {
+                    val ideArchive = configurationContainer.detachedConfiguration(dependency).singleFile
```

Comment:
Unfortunately, `detachedConfiguration` have weird names in the UI (see https://github.com/gradle/gradle/issues/17095), so you might be better creating a named configuration instead, so the user knows better what they download

Reply:
That's a good point, fixed. Thanks for the review!